squad/core/templates/squad/notification/failed_test_jobs.html

```diff
@@ -7,7 +7,7 @@
     <li>Project: {{project}}</li>
     <li>
         Build:
-        <a href="{{settings.BASE_URL}}/{{project.slug}}/build/{{build.version}}/">{{settings.BASE_URL}}/{{project.slug}}/build/{{build.version}}/</a>
+        <a href="{{settings.BASE_URL}}/{{project.group.slug}}/{{project.slug}}/build/{{build.version}}/">{{settings.BASE_URL}}/{{project.slug}}/build/{{build.version}}/</a>
```

Comment:
you missed the link text part

Reply:
but the link would have worked :) Fixed now.